lib/git_reflow/git_server/pull_request.rb

```diff
@@ -35,13 +53,48 @@ def reviewers_pending_response
         reviewers - approvals
       end
 
+      def check_approvals
+        has_comments_or_approvals = (has_comments? or approvals.any?)
+
+        case self.class.minimum_approvals
+        when "0"
+          true
+        when "", nil
+          # Approvals from every commenter
+          has_comments_or_approvals && reviewers_pending_response.empty?
+        else
+          approvals.size >= self.class.minimum_approvals.to_i and !last_comment.match(self.class.approval_regex).nil?
```

Comment:
so I just realized an issue with checking the last comment for `lgtm` in the context of the super class `PullRequest` is that this will now _always_ fail for Bitbucket unless there is a `lgtm` comment.  Bitbucket allows reviewers to "Approve" the review, making the check for anything in the comments (other than it's existence after an approval) pointless.

We'll need to pull this check out of here and make the last `else` statement only check for minimums.  Then we'll have to create a `check_approvals` method on the `GitReflow::GitServer::GitHub::PullRequest` that adds the extra feature.  Something like this:

```ruby
# In GitReflow::GitServer::Github::PullRequest
def check_approvals
  if self.class.minimum_approvals.to_i > 0
    approvals.size >= self.class.minimum_approvals.to_i and !last_comment.match(self.class.approval_regex).nil?
  else
    super
  end
end
```